### ISSUE TITLE
[BUG] In game API using Summoner Name instead of Riot Id

### ISSUE THREAD

**Issue by HeyPalmer** [RiotGames/developer-relations#857]
**Bug Description**  
The In game API does not currently support Riot Ids

**Problem Description**  

Using one of the following only yields summoner name without the tagline as needed for the Riot Id

https://127.0.0.1:2999/liveclientdata/allgamedata
or
https://127.0.0.1:2999/liveclientdata/playerlist

**Expected Result**  
/liveclientdata/playerlist should include Riot Id

**Actual Result**  
Only Summoner Name is available from the result.

**Repro Steps**  
Start a game. Go to /liveclientdata/playerlist. Find only a "summonerName" field but not one for "tagLine" or "riotId"

**Comment by MWstudio** [comment]
liveclientdata/eventdata too.

**Comment by bonepl** [comment]
I bump this. activePlayer(Name) webservices has different summonerName than playerList

**Comment by tisbells** [comment]
Hey there. Developer Relations doesn't own the in game API, but we've raised this and if we hear more information we'll provide an update.

**Comment by meesvandongen** [comment]
The response changed to include the `#tag`. However, `allPlayers` was not updated yet unfortunately. Would be great if that can still be fixed.

**Comment by MWstudio** [comment]
> The response changed to include the `#tag`. However, `allPlayers` was not updated yet unfortunately. Would be great if that can still be fixed. Although we can already split by `#` and tell with relative certainty that two players are equal.

what api been changed? i can't find `#tag` data any apis

**Comment by MWstudio** [comment]
> The response changed to include the `#tag`. However, `allPlayers` was not updated yet unfortunately. Would be great if that can still be fixed. Although we can already split by `#` and tell with relative certainty that two players are equal.
`events` was not updated too

**Comment by MWstudio** [comment]
https://127.0.0.1:2999/liveclientdata/playerlist
https://127.0.0.1:2999/liveclientdata/eventdata
and others needs tagline

**Comment by xEcEz** [comment]
Would be great to see an update about this. Thanks.

**Comment by tisbells** [comment]
Hey there - We sent a follow-up on this to the team earlier this week, but unfortunately it's unlikely we'll hear any updates until after the new year. If something changes I'll let y'all know

**Comment by JoaoDanielRufino** [comment]
Hi @tisbells, any updates on this ?

**Comment by tisbells** [comment]
> Hi @tisbells, any updates on this ?

Unfortunately no updates yet - This work is still being prioritized. I will check in with them and provide more info as I get it

**Comment by MWstudio** [comment]
Hi @tisbells , this issue is forgotten from riot?

**Comment by tisbells** [comment]
Hey there - Apologies for the long wait on this. Progress was made this week and you should see some updates for this in PBE today and I believe live with patch 14.10 on May 14. If something changes, I'll provide another update.

**Comment by MWstudio** [comment]
@tisbells thank you for update. i checked some apis, but only /liveclientdata/eventdata api return summoner name without tagline

**Comment by tisbells** [comment]
@MWstudio Thanks for letting us know - Are you seeing that in PBE? And are the others returning the Riot ID as expected? Here is a list of the APIs that were supposed to be updated:
```
"/liveclientdata/activeplayername"
"/liveclientdata/playerlist"
"/liveclientdata/gamestats"
"/liveclientdata/playeritems"
"/liveclientdata/playerscores"
"/liveclientdata/playersummonerspells"
"/liveclientdata/playermainrunes"
"/liveclientdata/activeplayerrunes"
"/liveclientdata/allgamedata"
"/liveclientdata/eventdata" (I will let them know this one is still missing the Riot ID)
"/liveclientdata/activeplayer"
"/liveclientdata/activeplayerabilities"

```
You should see fields in the response for `riotId`, `riotIdGameName`, and `riotIdTagLine`

**Comment by MWstudio** [comment]
@tisbells yes i tested it in PBE Server. other APIs return riotId, riotIdGameName, and riotIdTagLine. but only "/liveclientdata/eventdata" is not updated

**Comment by tisbells** [comment]
Thanks for confirming, I've let them know!

**Comment by AoshiW** [comment]
could you also update the documentation at https://developer.riotgames.com/docs/lol#game-client-api_live-client-data-api? It still uses `summonerName` for the query parameters.
